data/schema.ttl

```diff
@@ -9570,8 +9570,9 @@ we define a supporting type, [[SpeakableSpecification]]  which is defined to be
         :VideoGame,
         :VideoGameSeries,
         :VideoObject ;
-    :rangeIncludes :Person ;
-    rdfs:comment "An actor, e.g. in TV, radio, movie, video games etc., or in an event. Actors can be associated with individual items or with a series, episode, clip." .
+    :rangeIncludes :Person,
+        :PerformingGroup ;
```

Comment:
How about also adding :PerformanceRole here to make the JSON-LD usage example consistent with schema? (See data/examples.txt LL10845-10860)

Reply:
Role is meant to be applicable to any property, not just actor, as per the [post announcing PerformanceRole in 2014](http://blog.schema.org/2014/06/introducing-role.html)

Reply:
Oh I see! Sorry for missing that. I'll drop the pull request.

Reply:
Oh I see! Sorry for missing that, and thanks for the explanation!